framework/src/SyncObj.cpp

```diff
@@ -38,7 +38,7 @@
 #include "DriverFramework.hpp"
 #include "SyncObj.hpp"
 
-#if defined(__PX4_POSIX) || defined(__PX4_QURT)
+#if defined(__PX4_POSIX_SITL)
```

Comment:
Is this a new define?

Reply:
Yes.
https://github.com/PX4/Firmware/pull/10648/commits/ad81253054a81d6c09ac3b38d95479b91fcd806a

Reply:
Ok. Can't we always define it for `__PX4_POSIX` as `px4_pthread_cond_timedwait` and the Firmware will then map it to the right thing?